scripts/build-doc

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
+
+/* globals require process */
```

Comment:
😄  i saw that in your `graphql` branch. replacing it now.
i doubt the file is being "auto" linted tho. no complains when that line is removed.